feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/MapboxMap.kt

```diff
@@ -24,8 +25,8 @@ import com.mapbox.maps.plugin.scalebar.scalebar
 fun MapboxMap(state: HomeState) {
     val mapBoxView = mapView(state.mapBoxPublicKey, state.mapBoxStyleUri)
 
-    var initialLocationReceived by rememberSaveable { mutableStateOf(false) }
-    LaunchedEffect(Unit) { initialLocationReceived = false }
+    var initialUserLocationReceived by rememberSaveable { mutableStateOf(false) }
```

Comment:
Do we need this variables now?
I assume we can remove everything

Reply:
Ok, removed